gui/src/daemon/model.rs

```diff
@@ -203,10 +203,11 @@ impl SpendTx {
 
     /// Feerate obtained if all transaction inputs have the maximum satisfaction size.
     pub fn min_feerate_vb(&self) -> Option<u64> {
-        // This assumes all inputs are internal (have same max satisfaction size).
-        let max_tx_vbytes =
-            self.psbt.unsigned_tx.vsize() + (self.max_sat_vbytes * self.psbt.inputs.len());
-        self.fee_amount.map(|a| a.to_sat() / max_tx_vbytes as u64)
+        self.fee_amount.map(|a| {
+            a.to_sat()
+                .checked_div(self.max_vbytes)
+                .expect("max_vbytes should be greater than zero")
```

Comment:
nit: Hopefully, yes! But why? I think that's the info which should go here. For instance "a PSBT with an empty unsigned_tx would never deserialize therefore this is never 0" or simply "a descriptor's satisfaction size is never 0".

Reply:
I've now changed this to the second suggestion.